src/oic/oic/consumer.py

```diff
@@ -461,3 +513,40 @@ def check_session(self):
 
     def end_session(self):
         pass
+
+    # LOGOUT related
+
+    def backchannel_logout(self, request=None, request_args=None):
```

Comment:
As a new method, this should be annotated. You can then probably drop the second part of the docstring.

Reply:
OK